City-of-Helsinki/drupal-module-helfi-tpr
# Refactor unit migration to fetch data from individual pages, better translation support

To test this:

- drush pmu helfi_tpr
- drush en helfi_tpr
- drush mim tpr_unit --limit=100 --update

Go to Content -> Tpr unit and make sure they have translations.

You might have to change form/display settings to show fields (Structure -> Tpr unit -> Manage form display)
